dashboard/src/pages/ChapterDetails.vue

```diff
@@ -0,0 +1,242 @@
+            </template>
+            </FormControl>
+            <FormControl
+                :type="'url'"
+                size="md"
+                v-model="chapter.doc.instagram"
+                label="Instagram"
+            >
+            <template #prefix>
+                <svg  xmlns="http://www.w3.org/2000/svg"  width="24"  height="24"  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round"  class="w-5 text-gray-800 icon icon-tabler icons-tabler-outline icon-tabler-brand-instagram"><path stroke="none" d="M0 0h24v24H0z" fill="none"/><path d="M4 4m0 4a4 4 0 0 1 4 -4h8a4 4 0 0 1 4 4v8a4 4 0 0 1 -4 4h-8a4 4 0 0 1 -4 -4z" /><path d="M12 12m-3 0a3 3 0 1 0 6 0a3 3 0 1 0 -6 0" /><path d="M16.5 7.5l0 .01" /></svg>
+            </template>
+            </FormControl>
+            <FormControl
+                :type="'url'"
+                size="md"
+                v-model="chapter.doc.mastodon"
+                label="Mastodon"
+            >
+            <template #prefix>
+                <svg  xmlns="http://www.w3.org/2000/svg"  width="24"  height="24"  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round"  class="w-5 text-gray-800 icon icon-tabler icons-tabler-outline icon-tabler-brand-mastodon"><path stroke="none" d="M0 0h24v24H0z" fill="none"/><path d="M18.648 15.254c-1.816 1.763 -6.648 1.626 -6.648 1.626a18.262 18.262 0 0 1 -3.288 -.256c1.127 1.985 4.12 2.81 8.982 2.475c-1.945 2.013 -13.598 5.257 -13.668 -7.636l-.026 -1.154c0 -3.036 .023 -4.115 1.352 -5.633c1.671 -1.91 6.648 -1.666 6.648 -1.666s4.977 -.243 6.648 1.667c1.329 1.518 1.352 2.597 1.352 5.633s-.456 4.074 -1.352 4.944z" /><path d="M12 11.204v-2.926c0 -1.258 -.895 -2.278 -2 -2.278s-2 1.02 -2 2.278v4.722m4 -4.722c0 -1.258 .895 -2.278 2 -2.278s2 1.02 2 2.278v4.722" /></svg>
+            </template>
+            </FormControl>
+        </div>
+    </div>
+</div>
+</template>
+<script setup>
+import { useRoute } from 'vue-router'
+import { createDocumentResource, FormControl, FileUploader, Toast, toast } from 'frappe-ui'
+import CityCommunityBranding from '@/components/CityCommunityBranding.vue'
+import FossClubBranding from '@/components/FossClubBranding.vue'
+import { ref } from 'vue'
+const route = useRoute()
+
+const chapter = createDocumentResource({
+    doctype: 'FOSS Chapter',
+    name: `${decodeURI(route.params.id)}`,
```

Comment:
Again, decodeURI shouldn't be required...

Reply:
changed